lib/AssemblyUtil/TypeToFasta.py

```diff
@@ -0,0 +1,134 @@
+
+
+import os
+import time as _time
+
+from installed_clients.MetagenomeUtilsClient import MetagenomeUtils
+from AssemblyUtil.AssemblyToFasta import AssemblyToFasta
+from installed_clients.baseclient import ServerError as _MGUError
+
+
+from shutil import copyfile
+
+class TypeToFasta:
+
+    def __init__(self, callback_url, scratch, wrkspc):
+        self.ws = wrkspc
+        self.scratch = scratch
+        self.callback_url = callback_url
+        self.mgu = MetagenomeUtils(callback_url)
```

Comment:
Again, better to instantiate with a token explicitly so that this code could be reused. Instantiating a client is cheap so moving it to a method isn't a big deal.